standup_notes/__main__.py

```diff
@@ -10,8 +10,7 @@
 
 EXT = '.standup-notes.txt'
 STANDUP_NOTES = os.path.join(os.environ.get("HOME"), 'Desktop/standup-notes')
-STANDUP_TEMPLATE = os.path.join(os.path.dirname(__file__), 'standup.template')
-
+STANDUP_TEMPLATE = 'standup.template'
```

Comment:
You'll notice I had to move the _standup.template_ file. I couldn't quite figure out how the packaging was working, so I just changed the location of the file to be at the top level and everything worked. Not ideal, but it works.